app/controllers/admin/admin_invitations_controller.rb

```diff
@@ -26,6 +26,8 @@ def invite_from_queue
   end
 
   def grant_invites_to_users
+    authorize :admin_invitation
+
```

Comment:
Is it out of scope to say this should be an asynchronous task ?  Its going to timeout.

Reply:
Yeah, definitely out of scope.